packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt

```diff
@@ -63,6 +68,32 @@ internal object AgpConfiguratorUtils {
     project.pluginManager.withPlugin("com.android.application", action)
     project.pluginManager.withPlugin("com.android.library", action)
   }
-}
 
+  fun configureNamespaceForLibraries(appProject: Project) {
+    appProject.rootProject.allprojects { subproject ->
+      subproject.pluginManager.withPlugin("com.android.library") {
+        subproject.extensions.getByType(AndroidComponentsExtension::class.java).finalizeDsl { ext ->
+          if(ext.namespace == null){
+            val android = subproject.extensions.getByType(LibraryExtension::class.java)
+            val manifestFile = android.sourceSets.getByName("main").manifest.srcFile
+
+            if (manifestFile.exists()) {
+              val packageName = getPackageNameFromManifest(manifestFile)
```

Comment:
Can you null check `packageName` and set it only if it's not null?

Reply:
This will not work if the 3rd party librariy's name is before the 'app', such as 'aliyun' .....,
How can I fix it? @gabrieldonadel ,@cortinico